frontend/www/js/omegaup/components/submissions/List.vue

```diff
@@ -0,0 +1,154 @@
+<template>
+  <div class="row">
+    <div class="page-header">
+      <h1 class="text-center">
+        {{ T.submissionsListTitle }}
+      </h1>
+    </div>
+    <div class="panel panel-default">
+      <div class="panel-heading">
+        <h3 class="panel-title">
+          {{
+            UI.formatString(T.submissionsRangeHeader, {
+              lowCount: (page - 1) * length + 1,
+              highCount: page * length,
+            })
+          }}
+        </h3>
+      </div>
+      <div class="panel-body" v-if="shouldIncludeControls">
```

Comment:
qué tal

```suggestion
      <div class="panel-body" v-if="shouldShowControls">
```

para ser más consistentes??